gitindex/index.go

```diff
@@ -274,6 +274,8 @@ func SetTemplatesFromOrigin(desc *zoekt.Repository, u *url.URL) error {
 	}
 }
 
+const DefaultDeltaShardNumberFallbackThreshold uint64 = 150
```

Comment:
I picked this number out of thin air, but we can still control the fallback threshold with the `DELTA_SHARD_NUMBER_FALLBACK_THRESHOLD` env var.